qa/tasks/cephfs/filesystem.py

```diff
@@ -632,10 +636,14 @@ def create(self):
         log.debug("Creating filesystem '{0}'".format(self.name))
 
         self.mon_manager.raw_cluster_cmd('osd', 'pool', 'create',
-                                         self.metadata_pool_name, self.pgs_per_fs_pool.__str__())
+                                         self.metadata_pool_name,
+                                         '--pg_num_min', self.pg_num_min.__str__())
 
         self.mon_manager.raw_cluster_cmd('osd', 'pool', 'create',
-                                         data_pool_name, self.pgs_per_fs_pool.__str__())
+                                         data_pool_name,
+                                         '--pg_num_min', self.pg_num_min.__str__(),
+                                         '--target_size_ratio',
+                                         self.target_size_ratio.__str__())
```

Comment:
If using an ec pool, the target size ratio should be ~`.05` for the replicated data pool.

Reply:
Okay, I finally understoodd this logic.
Will fix it.